athena_cfn/README.md

```diff
@@ -0,0 +1,190 @@
+1. Check "I acknowledge that AWS CloudFormation might create IAM resources with custom names."
+1. Click "Update Stack"
+
+
+### C. Create and attach the new Quilt Policy
+
+After the `athena-cfn.yml` stack is created, you will need to:
+
+1. Go to the "Outputs" tab of the installed template
+1. Copy the `AthenaPolicy` ARN from the "Value" column
+1. Go back to "Admin Settings" in your Quilt Repository
+1. Scroll down to "Policies" and click "+" create a new one
+1. Enter the friendly name, e.g. "QuiltAthenaAccess"
+1. Check "Manually set ARN instead of configuring per-bucket permissions"
+1. Paste in the ARN from step (2)
+1. Click "Attach current policy to roles..."
+1. Select the Role you created in Section I.
+1. Click "Create"
+
+Note that it may take a few minutes for the policy credential cache to update.
+
+### D. Grant Quilt access to the new Bucket
+
+As with (B), you will need to:
+
+1. Go to the "Outputs" tab of the installed template
+1. Copy the `OutputBucket` value
+1. Go back to "Admin Settings" in your Quilt Repository
+1. Go to "Buckets" tab and click "+" create a new one
+1. Paste the value from (2) into "Name"
+1. Add a user-friendly description, e.g. "Athena Query Output"
+1. Click "Add"
+
+## IV. Enable Athena for Individual Buckets
+
+Note that this process assumes you have already:
+* added the relevant S3 bucket to the Quilt registry
+* added that bucket to a Policy for the Role from (I)
+* created at least one package in that bucket
+
+### A. Get link for athena-bucket-cfn
+
+Go to CloudFormation in the same Account and Region as before:
+
+1. Click dropdown "Create stack" > "With new resources (standard)"
+1. Select "Template is ready" (default)
+1. Select "Amazon S3 URL" (default)
+1. Paste in the `athena-bucket-cfn.yml` Object URL from before
+1. Click Next
+1. Enter a new "Stack name", e.g.: `your-bucket-athena-quilt`
+1. Enter "QuiltAthenaStack", e.g.: `athena-quilt` (from part III.A.7)
+1. Enter "QuiltBucket" to pull metadata from, e.g.: `your-bucket` (without `s3` or `arn` prefixes)
+1. Enter "QuiltBucketID", e.g.: `your_bucket` (with '-' replaced by '_')
+1. Click Next
+1. Click Next
+1. At the bottom, open the "Quick-create link" triangle
+1. Click "**Open quick-create link**" to get a wizard that makes it easy to
+rerun this for multiple buckets.
+
+Save this link in a shared space so you and other AWS users can easily index additional Quilt buckets.
+
+### B. Use quick-create link for each bucket you want to index
+
+1. Open the link from (A.13)
+1. Update the stack name and bucket parameters to match THIS bucket
+1. Click "Create change set"
+1. Wait until "Status = CREATE_COMPLETE" (may need to refresh using upper-right "cycle" icon)
+1. Review Changes
+1. Click "Execute" in upper right
+1. Select "Roll back all stack resources" (default)
+1. Click "Execute change set"
+
+### C. Manually invoke view-creating queries
+
+Unfortunately, CloudFormation does not automatically generate the proper views.
```

Comment:
I think we should say that this is "overly complicated" rather than impossible? https://stackoverflow.com/questions/56289272/create-aws-athena-view-programmatically#56347331

Reply:
It's also outside the scope of the spirit of CloudFormation which is infra, not so much business logic like views.